modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -133,3 +137,15 @@ if(DEFINED uicr_path)
     zephyr_library_compile_definitions(CONFIG_GPIO_AS_PINRESET)
   endif()
 endif()
+
+if(CONFIG_SOC_NRF54L15)
+	dt_prop(clock_frequency PATH "/cpus/cpu@0" PROPERTY "clock-frequency")
+	math(EXPR clock_frequency_mhz "${clock_frequency} / 1000000")
+	zephyr_compile_definitions("CONFIG_CPU_FREQ_MHZ=${clock_frequency_mhz}")
+endif()
+
+zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_SKIP_CLOCK_CONFIG NRF_SKIP_CLOCK_CONFIGURATION)
+
+if(CONFIG_SOC_PLATFORM_NRF54L AND CONFIG_NRFX_PPIB)
+  zephyr_library_sources(${HELPERS_DIR}/nrfx_gppi_dppi_ppib_lumos.c)
```

Comment:
is "lumos" a public name?

Reply:
yes

Reply:
nevertheless I don't see it being used in other places. The platform is referred to as "nrf54l platform" or "nrf54lx platform"

Reply:
This is an nrfx file, so has to stay as is for now.